api-reference/10 UI Components/Errors and Warnings/W1020.md

```diff
@@ -3,7 +3,9 @@ id: ErrorsUIWidgets.W1020
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Occurs if the [shading](/Documentation/ApiReference/UI_Components/dxPopover/Configuration/#shading) property is **false**. [hideEvent](/Documentation/ApiReference/UI_Components/dxPopover/Configuration/hideEvent/) is ignored.
```

Comment:
It's unclear how the `hideEvent` and `shading` properties are connected and why they depend on each other.